index.js

```diff
@@ -171,10 +442,34 @@
     }
   });
 
-  //  toMaybe :: a? -> Maybe a
+  //# toMaybe :: a? -> Maybe a
+  //.
+  //. Takes a value and returns Nothing if the value is null or undefined;
+  //. Just the value otherwise.
+  //.
+  //. ```javascript
+  //. > S.toMaybe(null)
+  //. Nothing()
+  //.
+  //. > S.toMaybe(42)
+  //. Just(42)
+  //. ```
   S.toMaybe = R.ifElse(R.isNil, Nothing, Just);
 
-  //  encase :: (* -> a) -> (* -> Maybe a)
+  //# encase :: (* -> a) -> (* -> Maybe a)
+  //.
+  //. Takes a function `f` which may throw and returns a curried function
+  //. `g` which will not throw. The result of applying `g` is determined by
+  //. applying `f` to the same arguments: if this succeeds, `g` returns Just
+  //. the result; otherwise `g` returns Nothing.
+  //.
+  //. ```javascript
+  //. > S.encase(eval)('1 + 1')
+  //. Just(2)
+  //.
+  //. > S.encase(eval)('1 +')
+  //. Nothing()
+  //. ```
```

Comment:
@charleswli suggested encasing a function other than `JSON.parse`, since Sanctuary exports an encased version of that function. The new examples apply `eval` to syntactically valid and syntactically invalid input.